kingpin/actors/packagecloud.py

```diff
@@ -0,0 +1,256 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2014 Nextdoor.com, Inc
+
+"""
+:mod:`kingpin.actors.packagecloud`
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The packagecloud actor allows you to delete packages from a packagecloud repo
+or check for existance. The actor supports dry mode by looking for packages to
+delete, but leaving them intact.
+
+**Required Environment Variables**
+
+:PACKAGECLOUD_ACCOUNT:
+  packagecloud account name, i.e. https://packagecloud.io/accountname
```

Comment:
Also, you're example (or defaults) should be wrapped in stars to properly bring attention to them in the docs. See example here: https://github.com/Nextdoor/kingpin/blob/master/kingpin/actors/slack.py#L31